# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/tcsh/undef_tiocgltc.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This patch is needed on Alpha AXP ...

--- ./ed.h.orig	Sat Nov 25 09:42:59 2000
+++ ./ed.h	Sat Nov 25 09:43:03 2000
@@ -37,6 +37,10 @@
 #ifndef _h_ed
 #define _h_ed
 
+#ifdef TIOCGLTC
+# undef TIOCGLTC
+#endif
+
 #ifndef EXTERN
 # define EXTERN extern
 #endif
